content/StsServerIdentity/Resources/SharedResource.de-DE.resx

```diff
@@ -358,22 +358,22 @@
     <value>Dokumentation</value>
   </data>
   <data name="CONFIGURE_AUTHENTICATOR_APP_STEPS" xml:space="preserve">
-    <value>Führen Sie die folgenden Schritte aus, um eine Authenticator-App zu verwenden:</value>
+    <value>Führen Sie die folgenden Schritte aus, um eine Authentifizierungs-App zu verwenden:</value>
   </data>
   <data name="DELETE_PERSONAL_DATA" xml:space="preserve">
-    <value>Löschen Sie persönliche Daten</value>
+    <value>Persönliche Daten löschen</value>
   </data>
   <data name="DELETE_PERSONAL_DATA_AND_CLOSE" xml:space="preserve">
-    <value>Löschen Sie Daten und schließen Sie mein Konto</value>
+    <value>Daten löschen und mein Konto schliessen</value>

```

Comment:
Be aware: schliessen should be schlie­ßen
 https://www.duden.de/rechtschreibung/schlieszen

Reply:
@robertmuehsig @kdaveid ss is good in Switzerland. I'll merge this, create a de-DE with the ß. I don't think this will change much in the future, so we should be able to manage the changes in both files. 

Thanks for you help!

Reply:
Well, if we provide de-DE we should use ß. In de-CH we could change it. Thanks @robertmuehsig - I'll take it into account for the next update.